launcher/tasks/SequentialTask.cpp

```diff
@@ -33,11 +33,17 @@ void SequentialTask::executeTask()
 
 bool SequentialTask::abort()
 {
-    bool succeeded = true;
-    for (auto& task : m_queue) {
-        if (!task->abort()) succeeded = false;
+    if(m_currentIndex == -1 || m_currentIndex >= m_queue.size()) {
+        m_queue.clear();
+        return true;
```

Comment:
isn't this missing `emitAborted()`?

Reply:
I don't know actually. If it enters this `if`, it's either not running yet (and so `emitAborted` will do nothing), or it's already finished, in which case, most likely it will already exit the sequential task with whatever status it had, so aborting here would only emit the same "finished" signals twice. So idk really :/